tests/fixtures/transform_custom.py

```diff
@@ -29,3 +36,55 @@ def transform(X, transformer, y=None):
         )
     else:
         return pd.DataFrame(transformed)
+
+
+class CustomTask(TransformerInterface):
+    @classmethod
+    def load_task(cls, artifact_directory):
+        """
+        Helper method to abstract deserializing the pickle object stored within `artifact_directory` and
+        returning the custom task. Any variables that were excluded in `save_task` must be manually loaded
+        proceeding this function.
+
+        Parameters
+        ----------
+        artifact_directory: str
+            Path to the directory to save the serialized artifact(s) to.
+
+        Returns
+        -------
+        cls
+            The deserialized object
+        """
+        # only one artifact is expected in the folder: sklearn_transform.pkl or sklearn_transform_dense.pkl
+        artifact_file = glob.glob(os.path.join(artifact_directory, "sklearn_transform*.pkl"))[0]
```

Comment:
but how will it know which one to choose?  does it matter if it's dense or not?  if not, why not excplicitly choose one.

Reply:
During the test only artifact is copied into the code_dir.